bladex/profilebase.py

```diff
@@ -36,12 +48,20 @@ def __init__(self):
 
     def _update_edges(self):
         """
-        Private method that identifies and updates the airfoil's leading and trailing edges.
-
-        Given the airfoil coordinates from the leading to the trailing edge, if the trailing edge has a non-zero thickness, 
-        then the average value between the upper and lower trailing edges is taken as the true trailing edge, hence both 
-        the leading and the trailing edges are always unique.
+        Private method that identifies and updates the airfoil's leading and
+        trailing edges.
+
+        Given the airfoil coordinates from the leading to the trailing edge,
+        if the trailing edge has a non-zero thickness, then the average value
+        between the upper and lower trailing edges is taken as the true
+        trailing edge, hence both the leading and the trailing edges are always
+        unique.
         """
+        assert(self.xup_coordinates[0] == self.xdown_coordinates[0],
```

Comment:
Avoid `assert` statements, they are removed from code when the compiled code (using optimization flag) is generated. Replace with `if..raise`.